### PR TITLE
own info

### PR DESCRIPTION
**Pull request by CatalinPirvu**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://CatalinPirvu.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -348,3 +348,9 @@
   facebook: maitreyav
   twitter: maitvyas
   bio: 17, Second GCI, Design + Code, The web is what you make of it. 
+
+- name: Catalin Pirvu
+  github: https://github.com/CatalinPirvu
+  image: catalinpirvu.jpg
+  facebook: https://www.facebook.com/pirvu.catalin.98
```
Comment (jigyasa-grover) [inline _data/students.yml]: Hey @CatalinPirvu , You just need to add User IDs/Names in Facebook and GitHub. Follow the pattern, no need of complete URLs.

**Comment by CatalinPirvu** [comment]
ive put broken fb and github links. should i make another pull req?

**Comment by jigyasa-grover** [comment]
No, just rectify in here itself. A new PR ain't required :)

**Comment by CatalinPirvu** [comment]
is everything allright? :)

**Pull request merged**